demo/index.html

```diff
@@ -10,7 +10,7 @@
     <script src="../bower_components/jquery/dist/jquery.js"></script>
 
     <script src="../bower_components/angular/angular.js"></script>
-    <script src="//angular-ui.github.io/bootstrap/ui-bootstrap-tpls-0.9.0.js"></script>
+    <script src="http://angular-ui.github.io/bootstrap/ui-bootstrap-tpls-0.13.0.js"></script>

```

Comment:
Why you added explicit http protocol? it's better for browser negotiate protocol.